webapp/platform/client/src/websocket.test.ts

```diff
@@ -321,14 +360,16 @@ describe('websocketclient', () => {
             if (mockWebSocket.onclose) {
                 mockWebSocket.onclose();
             }
-            if ((mockWebSocket.close as jest.Mock).mock.calls.length > 2) {
-                client.close();
+            if ((mockWebSocket.close as jest.Mock).mock.calls.length === 3) {
```

Comment:
```suggestion
            if (jest.mocked(mockWebSocket.close).mock.calls.length === 3) {
```
You can check https://hub.mattermost.com/private-core/pl/hiek6sb6njfyxy9io7xxfskyqw if you want to know more why :P

Reply:
Done :+1: